soh/include/z64save.h

```diff
@@ -65,6 +65,7 @@ typedef struct {
     /*      */ u32 entrancesDiscovered[SAVEFILE_ENTRANCES_DISCOVERED_IDX_COUNT];
     /*      */ u32 scenesDiscovered[SAVEFILE_SCENES_DISCOVERED_IDX_COUNT];
     /*      */ u8 locationsSkipped[RC_MAX];
+    /*      */ bool foundGreg;
```

Comment:
Arguably something like randomizerInf would be a better fit for this bool, but I won’t die on that hill

Reply:
that makes sense! doing that makes it so I won't need to make save manager changes too!